app/javascript/controllers/nested_invites_controller.js

```diff
@@ -102,4 +99,30 @@ export default class extends Controller {
       })
     });
   }
+
+  sendInviteEmails(id, invitedAs, inviteId, data) {
+    if(data.errors.length > 0 && data.counter === 0) {
+       $.each(data.errors, function(index, error) {
+        toastr.error(error)
+      })
+    }
+    else if(data.errors.length > 0 && data.counter > 0) {
+      $.each(data.errors, function(index, error) {
+        toastr.error(error)
+      })
+      $.post(`/proposals/${id}/invites/${inviteId}/invite_email?invited_as=${invitedAs}`, function() {
+        setTimeout(function() {
+          window.location.reload();
+        }, 3000)
+      })
+    }
```

Comment:
Why are we sending invite emails from JavaScript instead of from the Rails controller, after a successful @invite.save? It seems an unnecessary complication to do a second post when the data has already been sent to the controller.